app/views/NewMessageView.js

```diff
@@ -213,9 +239,20 @@ class NewMessageView extends React.Component {
 	}
 
 	renderItem = ({ item, index }) => {
-		const { search, chats } = this.state;
+		const { search, chats, permissions } = this.state;
 		const { baseUrl, user, theme } = this.props;
 
+		let hasPermissions = false;
```

Comment:
`NewMessagesView` is used essential to display only Direct Message rooms, we don't need to check all these room types.